recipe/meta.yaml

```diff
@@ -8,43 +8,45 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4e913d807aa6598c320e8a50c50e2ee0602bc45240b485e3f8bc06f13060084c
+  sha256: 21f535de2e7a82c94c130c054adb5c7421d480d5619d61073996e2f89bcb879b
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  skip: true  # [py<39]
 
 requirements:
   host:
     - python
     - pip
     - pytest-runner
+    - setuptools
+    - setuptools_scm
+    - wheel
   run:
     - python
-    - pywin32  # [win]
+    - pywin32 >=226  # [win]
 
 test:
-  imports:
-    - portalocker
   source_files:
     - portalocker_tests
+  imports:
+    - portalocker
   requires:
-    - pytest >=3.4.0
-    - pytest-cache >=1.0
-    - pytest-cov >=2.5.1
-    - pytest-flakes >=2.0.0
-    - pytest-pep8 >=1.0.6
-  # TODO: how to skip just test_combined ?
-  # commands:
-  #  - pytest -k "not test_combined" portalocker_tests
+    - pip
+    - pytest
+    - redis
+  commands:
+    - pip check
+    - pytest -v portalocker_tests --ignore=portalocker_tests/test_combined.py
 
 about:
   home: https://github.com/WoLpH/portalocker
   license: PSF-2.0
   license_family: PSF
   license_file: LICENSE
   summary: Portalocker is a library to provide an easy API to file locking.
```

Comment:
```suggestion
  summary: Portalocker is a library to provide an easy API to file locking.
  description: |
    - An easy library for Python file locking. It works on Windows, Linux, BSD and Unix systems and can even perform distributed locking. Naturally it also supports the with statement.
```

Reply:
Forgot to push the description. Thank you!